app/views/publish/courses/course_button_panel/_withdrawn.html.erb

```diff
@@ -0,0 +1,3 @@
+<span class="govuk-hint govuk-!-font-size-16" data-qa="course__withdrawn_date">
+  Withdrawn on <%= l(course.withdrawn_at.to_date) %> at <%= course.withdrawn_at.strftime("%l:%M%P") %>
```

Comment:
Can we move the format in to translations

See https://guides.rubyonrails.org/i18n.html#adding-date-time-formats

Reply:
sorted